molecule/default/requirements.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- src: idealista.java_role
+  version: 5.1.0
```

Comment:
I know its only for molecule tests, but, there is a newer version of this role just in case you want to use this instead :) https://github.com/idealista/java_role/releases/tag/6.1.1